nixos/modules/services/logging/logstash.nix

```diff
@@ -152,6 +156,16 @@ in
         '';
       };
 
+      extraJvmOptions = mkOption {
+        type = types.lines;
+        default = "";
+        description = "Extra JVM options, one per line (jvm.options format).";
+        example = ''
+          -Xms2g
+          -Xmx2g
+        '';
+      };
+
 
     };
```

Comment:
```suggestion
      };
    };
```

Reply:
Thanks, fixed (I deleted only 1 newline to keep it balanced with the matching opening `{`).